tests/Asset/OMJob.php

```diff
@@ -3,10 +3,9 @@
 namespace SlmQueueDoctrineTest\Asset;
 
 use Doctrine\Common\Persistence\ObjectManager;
-use DoctrineModule\Persistence\ObjectManagerAwareInterface;
 use SlmQueue\Job\AbstractJob;
 
-class OMJob extends AbstractJob implements ObjectManagerAwareInterface
```

Comment:
This is used in a strategy. I guess we should add a note about this?

https://github.com/juriansluiman/SlmQueueDoctrine/search?q=ObjectManagerAwareInterface

Reply:
We should create a Interface inside SlmQueueDoctrine namespace or just put in README the information about the functions needed (`public function setObjectManager(ObjectManager $objectManager)` and `public function getObjectManager()`)?

Reply:
A note is fine I think. Most people will use DoctrineORMModule anyway I'm guessing.

Additionally, we should note that DoctrineORMModule is not a direct requirement anymore and people should require it in their root composer.json or a break occurs.

Reply:
Done. Please, review the note.